code/obj/item/organ.dm

```diff
@@ -1296,6 +1296,25 @@
 		else
 			return ..()
 
+/obj/item/skull/attack_self(mob/user as mob)
+	var/nerdlist = list()
+	for (var/mob/living/carbon/M in range(7, user))

```

Comment:
orange() includes current tile, which might be nice

Reply:
other way around, orange explicitly excludes the origin tile

Reply:
@iamgoofball darn, looks like erik was wrong when he said that then

Reply:
additionally it'd be more performant to loop a list of all humans and check dist on 'em than it is to loop an entire range(7)

Reply:
additionally it'd be more performant to loop a list of all humans and check dist on 'em than it is to loop an entire range(7)

Reply:
Goddamnit!!! I'll fix that.

Reply:
> 
> 
> additionally it'd be more performant to loop a list of all humans and check dist on 'em than it is to loop an entire range(7)

Goon only has global lists of mobs and clients, sadly.

Reply:
Corrected to orange, works correctly even if someone's on the same tile.

Reply:
Then make a humans global tbh